_includes/imaging-tutorial-etcher-banner.html

```diff
@@ -1,5 +1,6 @@
 {% include icon.html type="info" %}
 There's an easier way to image your SD card! Check out
 [the "Getting Started" guide](/docs/getting-started#step-2-copy-the-image-on-to-the-sd-card){: .alert-link }
-for a cross-platform guide.
+for a cross-platform guide. This tutorial isn't being maintained and may not
```

Comment:
I may actually do zips for the next release to take advantage of the etcher metadata feature (unless etcher releases tar/metadata support before we release the image).

Either way, since the statement "This tutorial isn't being maintained and may not..." is in the banner, it leads me to believe that "This" is the etcher tutorial mentioned in the same box rather than the Win32DiskImager tutorial.

IMHO, if we aren't going to maintain a page, then we should remove it. I would say fix the page if some step is wrong for now and we can remove the page when we are confident that etcher works for everyone.